main_integration_test.go

```diff
@@ -23,6 +23,12 @@ import (
 // and the setup command works as expected.
 func TestSetup(t *testing.T) {
 
+	// Mage build is needed to pull in the packages from package-storage
```

Comment:
I don't think this is correct.

`test` shouldn't build any code. That's why you've dependencies between targets (not sure if mage supports it) or you call it `build test`

Reply:
Updated to only use fetchPackageStorage instead to prevent the building of the binary.

Reply:
I'm afraid that this is something we won't skip.

Reply:
@mtojek Not sure I can follow your last comment. What part will we not skip?

Reply:
not to fetch packages